### PR TITLE
docs: changelog for 0919 stable release

### PR DESCRIPTION
**Pull request by MuyangAmigo**

### CODE REVIEW

**Review by kuojianlu — DISMISSED** [review]

File: packages/vscode-extension/CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,34 @@
 # Changelog
 
+## 5.2.0 - Sep 19, 2023
+
+This minor version update of Teams Toolkit addresses your requests with new features and bug fixes, including import exiting SharePoint Framework solution, new app tempaltes for Link Unfurling and AI Chat Bot and much more. We previously shared these incremental changes in the prerelease version and through a series of blog posts:
```
Comment (therealjohn) [inline packages/vscode-extension/CHANGELOG.md:5]: "This minor version update of Teams Toolkit includes new features and bug fixes based on your feedback. The new features include the ability to import an existing SharePoint Framework solution, new project templates for Link Unfurling and AI Chat Bot, and more. We previously shared these incremental changes in the prerelease version and through a series of blog posts:"

File: packages/vscode-extension/CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,34 @@
...
+- [July Prerelease](https://devblogs.microsoft.com/microsoft365dev/teams-toolkit-for-visual-studio-code-update-july-2023): Import an existing SharePoint Framework solution, implement link unfurling in Teams and Outlook.
+- [August Prerelease](https://devblogs.microsoft.com/microsoft365dev/teams-toolkit-for-visual-studio-code-update-with-new-ai-chat-bot-template/): Create an AI chat bot with the new AI Chat Bot template.
+
+We've listened to your feedback and included these additional new features, enhancements, and bug fixes to this release.
+
+New features:
+
+- Import an existing SharePoint Framework solution and continue development with Teams Toolkit.
+    ![SPFx Existing App](https://github.com/OfficeDev/TeamsFx/assets/11220663/3944f5c8-6c8c-4b4d-8df8-dc4f45b5967f)
+- A new link unfurling app template to help you get started with displaying rich content from links in Teams messages and Outlook emails.
```
Comment (therealjohn) [inline packages/vscode-extension/CHANGELOG.md:16]: "A new link unfurling project template to help you get started with displaying rich content from links in Teams messages and Outlook emails."

File: packages/vscode-extension/CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,34 @@
...
+    ![Link Unfurling](https://github.com/OfficeDev/TeamsFx/assets/11220663/6e8b982a-0531-4ec1-8420-f6f17955ff40)
+- A new app template `AI Chat Bot` to help you get started with building a GPT-like chat bot with AI capabilities using `Teams AI Library`.
```
Comment (therealjohn) [inline packages/vscode-extension/CHANGELOG.md:18]: "A new AI Chat Bot project template to help you get started with building a GPT-like chat bot with AI capabilities using the [Teams AI Library](https://github.com/microsoft/teams-ai)."

File: packages/vscode-extension/CHANGELOG.md [hunk 4]
```diff
@@ -1,5 +1,34 @@
...
+  ![AI Bot](https://github.com/OfficeDev/TeamsFx/assets/11220663/86a90d2a-efc3-4d8b-9e8c-5d34a1e8c081)
+- Onboarded a new sample `One Productivity Hub using Graph Toolkit with SPFx` that shows you how to build a tab for viewing your calendar events, to-do tasks and files by using Microsoft Graph Toolkit components and SharePoint provider.
```
Comment (therealjohn) [inline packages/vscode-extension/CHANGELOG.md:20]: "The Sample Gallery has a new sample, One Productivity Hub using Graph Toolkit with SPFx, that shows you how to build a Tab for viewing your calendar events, to-do tasks, and files using Microsoft Graph Toolkit components and a SharePoint provider."

File: packages/vscode-extension/CHANGELOG.md [hunk 5]
```diff
@@ -1,5 +1,34 @@
...
+    ![SPFx Sample](https://github.com/OfficeDev/TeamsFx/assets/11220663/084ac508-49ea-4b30-854c-8b4d578ff6ee)
+- Added CodeLens to the `teamsapp.yml` file to help you run life-cycle commands easily after editing the file.
```
Comment (therealjohn) [inline packages/vscode-extension/CHANGELOG.md:22]: "Run life-cycle commands like Provision, Deploy, and Publish using new CodeLens hints added in-line to `teamsapp.yml` and `teamsapp.local.yml` when editing the file."

File: packages/vscode-extension/CHANGELOG.md [hunk 6]
```diff
@@ -1,5 +1,34 @@
...
+    ![Inline Commands](https://github.com/OfficeDev/TeamsFx/assets/11220663/f6897b26-0e3c-441c-b028-32093e8322a7)
+
+Bug fixes:
+
+- Fixed an issue where the `Preview` tag for the `AI Chat Bot` app template was not included in previous version. ([#9901](https://github.com/OfficeDev/TeamsFx/pull/9901))
```
Comment (therealjohn) [inline packages/vscode-extension/CHANGELOG.md:27]: Prefer "project template" over "app template"

File: packages/vscode-extension/CHANGELOG.md [hunk 7]
```diff
@@ -1,5 +1,34 @@
...
+- Fixed an issue where sometimes Teams Toolkit may fail to activate due to an unexpected package dependency on `types/keytar`. ([#9910](https://github.com/OfficeDev/TeamsFx/pull/9910))
+
+More details on other enhancements, bug fixes and updates on Teams Toolkit CLI and TeamsFx SDK can be found from the blogs.
```
Comment (therealjohn) [inline packages/vscode-extension/CHANGELOG.md:30]: Remove this line.

**Review by therealjohn — CHANGES_REQUESTED** [review]

**Review by eriolchan — APPROVED** [review]

**Pull request merged**